docs/src/pages/components/drawers/drawers.md

```diff
@@ -7,7 +7,7 @@ components: Drawer, SwipeableDrawer
 
 <p class="description">Navigation drawers provide access to destinations in your app. Side sheets are surfaces containing supplementary content that are anchored to the left or right edge of the screen.</p>
 
-[Navigation drawers](https://material.io/design/components/navigation-drawer.html) provide access to destinations and app functionality, such as switching accounts. They can either be permanently on-screen or controlled by a navigation menu icon.
+[Navigation drawers](https://material.io/design/components/navigation-drawer.html) (or "sidebars") provide access to destinations and app functionality, such as switching accounts. They can either be permanently on-screen or controlled by a navigation menu icon.
```

Comment:
I feel like we handle aliases in different ways at the moment, there might be a piece of work to look at how we currently handle them and make them all consistent.